File: Map/Z_InterpretorWCDL/src/view/RunExample.java
package view;

import controller.Controller;
import controller.ControllerException;

public class RunExample extends Command 
{

	private Controller ctrl;
	
	public RunExample(String k, String desc, Controller control) 
	{
		super(k, desc);
		ctrl = control;
	}

	@Override
	public void execute() 
	{
		try
		{
			ctrl.allStep();
		}
		catch(ControllerException ce)
		{
			System.out.println(ce);
		}
		
	}

}

File: Map/Map/Map/view/RunExample.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Map.controller;

namespace Map.view
{
    class RunExample : Command
    {
        private Controller ctrl;

        public RunExample(string k, string desc, Controller c) : base(k, desc)
        {
            ctrl = c;
        }

        public override void execute()
        {
            try
            {
                ctrl.allSteps();
            }
            catch(Exception e)
            {
                Console.WriteLine(e.Message);
            }
        }
    }
}
